packages/render-fidelity-tools/src/components/renderers/babylon-viewer.ts

```diff
@@ -93,6 +94,9 @@ export class BabylonViewer extends LitElement {
             target.z),
         this[$scene]);
     camera.attachControl(this[$canvas]!, true);
+    // in babylon, camera use VERTICAL_FIXED mode by default, so fov here is
+    // equal to vertical fov
+    camera.fov = this[$degToRadians](verticalFoV);
```

Comment:
This solves the transformation errors of the latern and the flightHelment scenarios. BJS uses 0.8 radians as default fov value.  And for our config, we use 45 deg( about 0.78 radians).  that tiny difference has an obvious on these vertical scenarios. That's really silly of me to not using the accurate radians